resdk/resolwe.py

```diff
@@ -70,7 +71,7 @@ def _version_string_to_int(self, version):
         version_numbers = [int(number_string) for number_string in version.split(".")]
```

Comment:
On a more general note, this function looks like it was copied from [django-versionfield package](https://github.com/tonioo/django-versionfield/blob/master/versionfield/utils.py#L7-L40). Why not just require that package?